06-Modelling-Rigid-Bodies/01-Introduction-to-Rigid-Bodies.ipynb

```diff
@@ -254,11 +254,11 @@
        "  <rdf:RDF xmlns:dc=\"http://purl.org/dc/elements/1.1/\" xmlns:cc=\"http://creativecommons.org/ns#\" xmlns:rdf=\"http://www.w3.org/1999/02/22-rdf-syntax-ns#\">\n",
```

Comment:
<p>change</p><blockquote>place the central particle of each body first and set the <code>body</code> property of all members to the index&nbsp;of the constituent particle.</blockquote><p>to</p><blockquote>place the central particle of each body first and set the <code>body</code> property of all member particles (central <em>and</em> constituent) to the index&nbsp;of the central particle.</blockquote>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/hoomd-examples/pull/131/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='53'/>